config/jobs/gardener-extension-registry-cache/gardener-extension-registry-cache-build-images.yaml

```diff
@@ -0,0 +1,47 @@
+postsubmits:
+  gardener/gardener-extension-registry-cache:
+  - name: post-gardener-extension-registry-cache-build-images
+    cluster: gardener-prow-trusted
+    branches:
+    - ^main$
+    annotations:
+      description: Gardener extension registry-cache image build on main branch
+      testgrid-dashboards: gardener-extension-registry-cache
+      testgrid-days-of-results: "60"
+    decorate: true
+    max_concurrency: 1
+    spec:
+      serviceAccountName: image-builder
+      containers:
+      - name: image-builder
+        image: eu.gcr.io/gardener-project/ci-infra/image-builder:v20220908-8116cd8
+        command:
+        - /image-builder
+        args:
+        - --log-level=info
+        - --docker-config-secret=gardener-prow-gcr-docker-config
+        - --registry=eu.gcr.io/gardener-project/gardener-extension-registry-cache
```

Comment:
Other extensions use the following image repos:
`eu.gcr.io/gardener-project/gardener/extensions/provider-aws` and `eu.gcr.io/gardener-project/gardener/extensions/admission-aws`

We could follow this pattern by specifying `--registry=eu.gcr.io/gardener-project/gardener/extensions/registry-cache` and renaming our Dockerfile targets, correct?

Reply:
Done in 35cd955 and https://github.com/gardener/gardener-extension-registry-cache/pull/2

Reply:
`eu.gcr.io/gardener-project/gardener/extensions` looks like an orphan to me.
There are only a couple of images pushed to the subfolders and the youngest one is more than 1 year old

Reply:
What do you mean?
The provider extensions, networking extensions, shoot service extensions all push their images to this repo.
E.g., `eu.gcr.io/gardener-project/gardener/extensions/{provider,admission}-aws:v1.39.0-dev-a75ee7a8a5eaa01567150c8ec51f12379c39d14d` have been pushed very recently.